lib/Runtime/Library/ScriptFunction.cpp

```diff
@@ -409,10 +403,12 @@ namespace Js
             }
         }
 
+        Var computedNameVar = this->GetComputedNameVar();
+
         ENTER_PINNED_SCOPE(JavascriptString, computedName);
-        computedName = this->GetComputedName();
-        if (computedName != nullptr)
+        if (computedNameVar != nullptr)
         {
+            computedName = this->GetComputedName(computedNameVar, scriptContext);
```

Comment:
Nit: I think ScriptFunction::GetComputedName is clearer for calling a static

Reply:
Done